### PR TITLE
Redesign landing page

### PR DESCRIPTION
**Pull request by maggarwal99**
Landing page redesign

Follow-up action items:
We should add a "quick play" mode to allow players to play random/custom prompts.

Screen Shot 2022-07-13 at 8 00 59 PM
Screen Shot 2022-07-13 at 8 00 20 PM
Screen Shot 2022-07-13 at 7 59 32 PM
Screen Shot 2022-07-13 at 7 58 47 PM

### CODE REVIEW

**Comment by mliu59** [comment]
Will review soon, I think @danielliu04 said he wanted to try setting up the quick play thing. I'll create an issue and assign

**Comment by dqian3** [comment]
Looks great, only note is editor's pick should probably be renamed archive for now

**Review by dqian3 — CHANGES_REQUESTED** [review]
Please add the new logo I sent

**Comment by mliu59** [comment]
Yee I love this new UI, same comment as Dan about the archive section

**Comment by maggarwal99** [comment]
Made minor changes to the logo to be more consistent with Wikipedia's typography, hopefully, Oliver doesn't get upset _/\_

Screen Shot 2022-07-15 at 3 14 53 AM

File: templates/home.html [hunk 1]
```diff
@@ -18,149 +35,152 @@
...
-                </div>
+    <!-- sidebar, which will move to the top on a small screen -->
+    <div class="col-sm-3">
+        <nav id="toc" data-toggle="toc" class="sticky-top"></nav>
+    </div>
+    <!-- main content area -->
+    <div class="col-sm-9">
+        <div class="card">
+            <div class="alert alert-info my-0" role="alert">
+                <h5 class="my-0">Prompt of the day</h5>
```
Comment (bricehalder) [inline templates/home.html:71]: nit: I think "Prompt of the Day" would be better as this is a [title](https://capitalizemytitle.com/)

**Pull request merged**